File: week1/mario.c
#include <cs50.h>
#include <stdio.h>

//        #
//       ##
//      ###
//     ####
//    #####
//   ######
//  #######
// ########

int main(void)
{
    int number;

    do 
    {
        number = get_int("Height: ");
    }
    while (number <= 0 || number > 8);

    for (int i = 0; i < number; i++)
    {
        for (int j = 0; j < number - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        for i in range(n):
            for j in range(n):
                if j < n - i - 1:
                    print(" ", end="")
                else:
                    print("#", end="")
            print("\n", end="")
        break